app/controllers/api/v1/participants_controller.rb

```diff
@@ -9,7 +9,9 @@ class ParticipantsController < Api::ApiController
       include ParticipantActions
 
       def change_schedule
-        perform_action(service_namespace: ::Participants::ChangeSchedule)
+        service = recorder(service_namespace: ::Participants::ChangeSchedule).new(params: params_for_recorder)
+        result = service.call
+        render json: serialized_response(result)
       end
```

Comment:
Has this changed from what `perform_action` is doing?

Reply:
I think I see where you're going. `perform_action` has not changed and continues to work for the other actions. The internal API for `ChangeSchedule` has changed however this looks to be the only placed where it is used

Reply:
This should also be available under the `/ecf` namespace
Whenever I've touched this part of the code I've ported as well under `/ecf` namespace

Reply:
It is already available under the /ecf namespace. the routing just maps here as at the moment the behaviour is identical